sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -358,10 +381,21 @@ def make_function(state, arg):
   if sys.version_info[0] == 2:
     func_code = state.stack[-1].value
     func = types.FunctionType(func_code, globals)
-  else:
+  else:  # Python 3.x
     func_name = state.stack[-1].value
     func_code = state.stack[-2].value
-    func = types.FunctionType(func_code, globals, name=func_name)
+    closure = None
+    if sys.version_info[1] >= 6:
+      if arg & 0x08:
```

Comment:
Comment on the magic 0x08?

Reply:
Added comments. Also added consuming opcode arguments from the stack.